### ISSUE TITLE
333eth.io Scam

### ISSUE THREAD

**Issue by Slavikys13** [MetaMask/eth-phishing-detect#2386]
I ask you to remove the site 333eth.io from the blacklist. The site 333eth.io is entered by mistake in the black list MetaMask. Dear support, I am an investor of this site. Unblock the site 333eth.io and put everything back in place. The site 333eth.io is on the blacklist erroneously due to competitors.

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/2374
